ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -365,6 +366,59 @@ def create_storage_cluster(self):
         logger.info("Sleep 10 second after click on 'create storage cluster'")
         time.sleep(10)
 
+    def configure_external_kms(self):
+        """
+        Configure KMS as per the option set in the config.
+        """
+        self.do_click(self.dep_loc["enable_external_kms"])
+        kms_provider = config.ENV_DATA.get("KMS_PROVIDER")
+        if kms_provider == "azure-kv":
+            if not config.ENV_DATA.get("platform") == "azure":
```

Comment:
no "else" - block, we will skip this method without any log and kms related settings done